package.json

```diff
@@ -128,6 +129,9 @@
     "build": "node --max-old-space-size=3072 ./scripts/build/build.js",
     "build-docs": "node ./scripts/build-docs.js",
     "check-deps": "node ./scripts/check-deps.js",
-    "spellcheck": "npx -p cspell@4.0.31 cspell {bin,scripts,src}/**/*.js {docs,website/blog,changelog_unreleased}/**/*.md"
+    "spellcheck": "cspell {bin,scripts,src}/**/*.js {docs,website/blog,changelog_unreleased}/**/*.md"
+  },
+  "resolutions": {
+    "simple-html-tokenizer": "0.5.7"
```

Comment:
I would very strongly favor removing the simple-html-tokenizer 0.5.7 resolutions entry from the new release and accepting the breaking change while we have the chance. This would reduce the chance of issues with an unsupported dependency version.

Adding references to the following comments and discussions:

- <https://github.com/prettier/prettier/pull/7443#issuecomment-579353091>
- <https://github.com/tildeio/simple-html-tokenizer/issues/74>

Reply:
The "breaking change" is actually a bug; https://github.com/emberjs/ember.js/issues/18530

If we unpin simple-html-tokenizer, Prettier's `glimmer` parser will throw on closing tags like `</Title>` and `</TITLE>`.

Or is there more to it?

Reply:
I've opened a PR to fix this issue in simple-html-tokenizer

Reply:
Here is the link: https://github.com/tildeio/simple-html-tokenizer/pull/81

Reply:
@dcyriller Am I right that this issue is the only reason to keep the version pinned? Or is there something else?

Reply:
Yes, it is the only reason.

Reply:
And now https://github.com/prettier/prettier/pull/7245 is ready